components/dashboard/frontend/src/app/components/Explorer.tsx

```diff
@@ -46,8 +46,6 @@ export class Explorer extends React.Component<Props, any> {
                 </Row>
                 <ExplorerLiveFeed/>
                 <small>

```

Comment:
We should also remove sorrounding tag

Reply:
Will be fixed ina different PR by Max, so I'll close this one